tests/functional/directory/test_admin.py

```diff
@@ -56,3 +61,21 @@ def test_election_leave_serveral_service_ids(self):
         self.assertEquals(2, len(election))
         self.assertEquals(200, election[service_ids[0]]["status"]["status"])
         self.assertEquals(200, election[service_ids[1]]["status"]["status"])
+
+    def test_has_base(self):
+        info = self.admin.has_base(
+            'meta2', account=self.account, reference=self.container)
+        for peer, meta in info.iteritems():
+            self.assertEquals(200, meta['status']['status'])
```

Comment:
We should not use `assertEquals` or `assertNotEquals` anymore. They have been deprecated and replaced by the same without the ending `s`.